editor/templates/editoritem/preview.html

```diff
@@ -20,31 +20,35 @@
     <body>
         <header role="banner">
             <h1 class="description">
-                PREVIEW of <a href="{{item.get_absolute_url}}">{{item.name}}</a>
-                in <a href="{{item.project.get_absolute_url}}">{{item.project.name}}</a>
+                <a href="{{item.get_absolute_url}}">{{item.name}}</a>
+                <span class="light-maroon">in</span>
+                <a href="{{item.project.get_absolute_url}}">{{item.project.name}}</a>
             </h1>
 
             <span class="licence">
             {% if item.licence %}
-                &copy;
-                <abbr title="{{item.licence.name}}">
+            <span class="light-maroon no-underline-link"> &copy; </span>
+              <!--  <abbr title="{{item.licence.name}}"> -->
                     {% if item.licence.url %}
-                        <a target="_blank" href="{{item.licence.url}}">{{item.licence.short_name}}</a>
+                        <a class="light-maroon no-underline-link" target="_blank" href="{{item.licence.url}}">{{item.licence.short_name}}</a>
                     {% else %}
-                        {{item.licence.short_name}}
+                        <span class="light-maroon no-underline-link">{{item.licence.short_name}}</span>
                     {% endif %}
-                </abbr>
+               <!--  </abbr> -->
             {% endif %}
             </span>
 
             <a href="{{item.get_absolute_url}}">Edit</a>
 
             <a href="{% editoritem_url 'download' item.rel_obj %}?scorm">Download</a>
 
-            <button class="btn" id="share" type="button">Share</button>
+            <span class="pointer-effect underline-text" id="share"> Share </span>
 
-            <span id="embed-url">
-                <code tabindex="0">{{SITE_ROOT}}{{embed_url}}</code>
+            <span  id="embed-url">
+                <span class="copied-text-style" id="copied-text"></span>
+                <span class="light-maroon" id="link-label">Click <span class="pointer-effect underline-text color-white" id="copy-text">here</span> to copy this link - </span>
```

Comment:
Text like "click here" isn't great for screen readers - you'd have to navigate out of the button to find out what it will do. So the clickable text should be "Copy this link"